docs/feature/policy.rst

```diff
@@ -16,7 +17,28 @@ in the module. For example, for ``minpy.numpy``, use ``minpy.numpy.set_policy``
 ::
 
     import minpy.numpy as np
-    import minpy.dispatch.policy as policy
-    np.set_policy(policy.OnlyNumPyPolicy())
+    from minpy import OnlyNumPyPolicy
+    np.set_policy(OnlyNumPyPolicy())
+
+To make life easier, MinPy can also change the policy of all imported MinPy mocking modules at the same time. Simply add the following two lines before computation. Note that ``minpy.set_global_policy`` only changes all **imported** MinPy mocking modules.
+
+::
+
+    import minpy.numpy as np
+    import minpy.numpy.random as random
+    import minpy
+    minpy.set_global_policy(minpy.OnlyNumPyPolicy())
+    # np and random is now in OnlyNumPyPolicy policy
+
+Besides using ``minpy.set_global_policy``, MinPy also offers a function ``minpy.set_default_policy`` to set default policy for importing mocking modules in the future. For example:
+
+::
+
+    import minpy
+    minpy.set_default_policy(minpy.OnlyNumPyPolicy())
+    import minpy.numpy as np
+    import minpy.numpy.random as random
+    # np and random is now in OnlyNumPyPolicy policy
+
+It is worth mentioning that ``np.set_policy``, ``minpy.set_global_policy`` and ``minpy.set_default_policy``  only accept instances of policy classes.
```

Comment:
I'm a little bit confused about this. What is the previous behavior? What's the influence of different orders of `minpy.set_policy` and `import`? I think the best way is to have only one `set_policy`. We could either choose your `set_default` or `set_global` to reduce confusion. What do you think?